### DISCUSSION TITLE
Is it possible to make a fork private?

### DISCUSSION THREAD

**Discussion by crispyoz** [community/community#128990] [Repositories] [answered]
### Select Topic Area

Question

### Body

I can't see how to do this so I expect it is not possible but thought it worth asking.

If I fork a public repository, is it possible to make my fork private?   Usually I will create a private clone and later merge the changes into a fork of the public repo, but it's cumbersome.  It's not that I want to hide my fork, just that if it WIP I had someone pull my fork and then complain it was broken.

**Comment by davevad93** [comment, marked as answer]
Hi @crispyoz ,

GitHub doesn't natively allow to make `forked` repository `private`, 

![000000](https://github.com/community/community/assets/94308234/64b72641-9bb5-4d99-b6c8-f7d7a3b2e470)

These are the only workarounds known to me:
1. [private_fork.md](https://gist.github.com/0xjac/85097472043b697ab57ba1b1c7530274)
2. [Import your project to GitHub](https://github.com/new/import)

**Reply by crispyoz** [reply]
Thanks David,

I hadn't seen those docs before, but this is basically what I have been doing.

**Reply by davevad93** [reply]
You're welcome @crispyoz , well it looks like you got it right from the start since it seems to be the only way to do it. 👍

**Comment by patrick-knight** [comment]
[This GitHub App ](https://github.com/orgs/community/discussions/133150) might make that flow easier @crispyoz
